expyfun/_experiment_controller.py

```diff
@@ -1143,13 +1248,13 @@ def wait_for_click_on(self, objects, max_wait=np.inf, min_wait=0.0,
             If ``timestamp==True``, returns a tuple (str, int, int, float)
             indicating the first valid button clicked and its timestamp (or
             ``(None, None, None, None)`` if no acceptable button was clicked
-            between ``min_wait`` and ``max_wait``). If ``timestamp==False``,
+            between `min_wait` and `max_wait`). If ``timestamp==False``,
```

Comment:
That's my understanding. Not sure why it matters, maybe single backticks get rendered as links sometimes.